.idea/scopes/Gradle_public_API.xml

```diff
@@ -0,0 +1,3 @@
+<component name="DependencyValidationManager">
+  <scope name="Gradle public API" pattern="!lib:*..*&amp;&amp;!test:*..*&amp;&amp;!src[gradle.internal-testing.main]:*..*&amp;&amp;!src[gradle.smoke-test.smokeTest]:*..*&amp;&amp;!src[gradle.soak.integTest]:*..*&amp;&amp;!src:*..*.internal.*&amp;&amp;!src:*..*.internal..*&amp;&amp;(src:org.gradle.||src:org.gradle.api..*||src:org.gradle.authentication..*||src:org.gradle.build..*||src:org.gradle.buildinit..*||src:org.gradle.caching..*||src:org.gradle.concurrent..*||src:org.gradle.deployment..*||src:org.gradle.external.javadoc..*||src:org.gradle.ide..*||src:org.gradle.includedbuild..*||src:org.gradle.ivy..*||src:org.gradle.jvm..*||src:org.gradle.language..*||src:org.gradle.maven..*||src:org.gradle.nativeplatform..*||src:org.gradle.normalization..*||src:org.gradle.platform..*||src:org.gradle.play..*||src:org.gradle.plugin.devel..*||src:org.gradle.plugin.repository..*||src:org.gradle.plugin.use..*||src:org.gradle.plugin.management..*||src:org.gradle.plugins..*||src:org.gradle.process..*||src:org.gradle.testfixtures..*||src:org.gradle.testing.jacoco..*||src:org.gradle.tooling..*||src:org.gradle.swiftpm..*||src:org.gradle.model..*||src:org.gradle.testkit..*||src:org.gradle.testing..*||src:org.gradle.vcs..*||src:org.gradle.work..*||src:org.gradle.workers..*||src:org.gradle.util..*||src:org.gradle.kotlin.dsl.||src:org.gradle.kotlin.dsl.precompile.)" />
```

Comment:
You have here `org.gradle.kotlin.dsl` in the list of things. Is this also something we'd need to add to the list of the public API below? Same for `org.gradle.kotlin.dsl.precompile`. Anything else that misses in the list?

Reply:
I discussed this with @eskatos and @bamboo and apparently we have `PublicKotlinDslApi` next to `PublicApi` where  these packages come from.

It would be good to update the code there so that it's not misleading, but that should probably be in a different PR.